src/test/java/uk/nhs/digital/nhsconnect/lab/results/fixtures/FhirFixtures.java

```diff
@@ -0,0 +1,183 @@
+
+        performer.addName().setText("Dr Claire Hanna");
+        performer.setGender(Enumerations.AdministrativeGender.FEMALE);
+
+        return performer;
+    }
+
+    public static Organization generatePerformingOrganisation() {
+        Organization performingOrganization = new Organization();
+
+        performingOrganization.setName("Some hospital");
+
+        Address address = new Address();
+        address.addLine("10 Long Avenue")
+                .addLine("London");
+        address.setPostalCode("ZXC456 9SH");
+
+        performingOrganization.setAddress(List.of(address));
+
+        Coding coding = new Coding().setCode("dept").setDisplay("Haematology");
+        CodeableConcept codeableConcept = new CodeableConcept().setCoding(List.of(coding));
+
+        performingOrganization.setType(List.of(codeableConcept));
+
+        return performingOrganization;
+    }
+
+    public static Practitioner generateRequester() {
+        Practitioner requester = new Practitioner();
+
+        requester.addName().setText("Dr Bob Hope");
+        requester.setGender(Enumerations.AdministrativeGender.MALE);
+
+        return requester;
+    }
+
+    public static Organization generateRequestingOrganization() {
+        Organization requestingOrganization = new Organization();
+
+        requestingOrganization.setName("Some GP");
+
+        Address address = new Address();
+        address.addLine("5 New Street")
+                .addLine("London");
+        address.setPostalCode("IOP 789");
+
+        requestingOrganization.setAddress(List.of(address));
+
+        return requestingOrganization;
+    }
+
+    public static Organization generateSpecimenCollectingOrganization() {
+        Organization specimenCollectingOrganization = new Organization();
+
+        specimenCollectingOrganization.setName("Some specimen collecting organization");
+
+        Address address = new Address();
+        address.addLine("20 Red Road")
+                .addLine("London");
+        address.setPostalCode("RTY 222");
+
+        specimenCollectingOrganization.setAddress(List.of(address));
+
+        return specimenCollectingOrganization;
+    }
+
+    public static Practitioner generateSpecimenCollector() {
+        Practitioner specimenCollector = new Practitioner();
+
+        specimenCollector.addName().setText("Mr Collector");
+
+        return specimenCollector;
+    }
+
+    public static Specimen generateSpecimen() {
+        Specimen specimen = new Specimen();
+
+        Identifier identifier = new Identifier();
+        identifier.setValue("CH000037KA");
+
+        specimen.setIdentifier(List.of(identifier));
+
+        specimen.setStatus(Specimen.SpecimenStatus.AVAILABLE);
+
+        Coding coding = new Coding().setDisplay("VENOUS BLOOD");
+        CodeableConcept codeableConcept = new CodeableConcept().setCoding(List.of(coding));
+
+        specimen.setType(codeableConcept);
+
+        final int YEAR = 2010;
```

Comment:
I feel these should be constants being uppercase? Or even the `receivedTime` for visibility, which we can then go on to use in other places